documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    plt.plot([log.num_trees for log in logs], [log.evaluation.accuracy for log in logs])
</pre><p> </p><p>You have stated that this is OOB / test data, but the only dataset I can see used so far is train_ds (from <code>model.fit(x=train_ds)</code>). Is this test data or training data?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>